File: 1114.cpp
#include <iostream>
using namespace std;
int main(){
while(true){
    int n;
    cin >> n;
    if(n != 2002){
        cout << "Senha Invalida" << "\n";
    }
    if(n == 2002){
        break;
    }
}
cout << "Acesso Permitido" << "\n";
}

File: 1114.java
import java.util.Scanner;
public class Main {
public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        while(true){
                int a = input.nextInt();
                if(a == 2002){
                        System.out.println("Acesso Permitido");
                        break;
                }
                else {
                        System.out.println("Senha Invalida");
                }
        }
}
}
